.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+#####################################################
+#
+# List of approvers for OpenTelemetry .NET Instrumentation
+#
+#####################################################
+#
+# Learn about membership in OpenTelemetry community:
+#  https://github.com/open-telemetry/community/blob/master/community-membership.md
+# 
+#
+# Learn about CODEOWNERS file format: 
+#  https://help.github.com/en/articles/about-code-owners
+#
+
+* @macrogreg @MikeGoldsmith @pjanotti
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/.github/CODEOWNERS do you WANT to list names or just do what is done here?

Reply:
Good point. We will have a few folks that are not approvers on .NET. I will add the team and other folks manually.